Core/GraphQL/GraphQLManager.swift

```diff
@@ -60,6 +60,9 @@ extension GraphQLManager: GraphQLManageable {
         ) { result in
             switch result {
             case .success(let graphQLResult):
+                if let errors = graphQLResult.errors {
+                    return completion(.failure(errors[0]))
```

Comment:
This could cause a crash if for some reason there is no error in the array. Safer would be like this:

```
 if let errors = graphQLResult.errors,
    let firstError = errors.first {
         return completion(.failure(firstError))
}
```

Reply:
Nice catch! thanks. I'll fix it.